docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
-        api_key="ANONYMOUS",
-        project_name="shared/pytorch-lightning-integration",
-        experiment_name="default",  # Optional,
-        params={"max_epochs": 10,
-                "batch_size": 32},  # Optional,
-        tags=["pytorch-lightning", "mlp"]  # Optional,
-    )
-
-Create the **Trainer** and pass **neptune_logger** to logger
-------------------------------------------------------------
-.. code-block:: python3
+    pip install neptune-client -U
 
-    from pytorch_lightning import Trainer
+From Conda:
 
-    model = CoolSystem()
-    trainer = Trainer(max_epochs=10, logger=neptune_logger)
+.. code-block:: bash
 
-    trainer.fit(model)
+    conda install neptune-client -c conda-forge
 
-Log additional information after the **.fit** loop ends
--------------------------------------------------------
+**Install PyTorch Lightning and torchvision**
 
-You can log additional metrics, images, model binaries or other things after the ``.fit`` loop is over.
-You just need to specify ``close_after_fit=False`` in the ``NeptuneLogger`` initialization.
+From PyPI:
 
-.. code-block:: python3
+.. code-block:: bash
 
-    neptune_logger = NeptuneLogger(
-        api_key="ANONYMOUS",
-        project_name="shared/pytorch-lightning-integration",
-        close_after_fit=False,
-        ...
-    )
+    pip install pytorch-lightning torchvision
 
-**Log test metrics**
+From Conda:
 
-.. code-block:: python3
+.. code-block:: bash
+
+    conda install pytorch-lightning -c conda-forge
+    conda install pytorch torchvision -c pytorch
 
-    trainer.test(model)
+Now, you have all dependencies installed. Let's move to the actual integration.
 
-**Log additional metrics**
+Step 1 - Necessary Imports
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+No a big deal here, just imports. Note ``pytorch_lightning`` at the bottom.
 
 .. code-block:: python3
 
-    from sklearn.metrics import accuracy_score
-    ...
-    accuracy = accuracy_score(y_true, y_pred)
+    import os
+
+    import torch
+    from torch.nn import functional as F
+    from torch.utils.data import DataLoader
+    from torchvision.datasets import MNIST
+    from torchvision import transforms
 
-    neptune_logger.experiment.log_metric('test_accuracy', accuracy)
+    import pytorch_lightning as pl
 
-**Log performance charts**
+Step 2 - Model Hyper-Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You will see them in Neptune parameters tab.
 
 .. code-block:: python3
 
-    from scikitplot.metrics import plot_confusion_matrix
-    import matplotlib.pyplot as plt
-    ...
-    fig, ax = plt.subplots(figsize=(16, 12))
-    plot_confusion_matrix(y_true, y_pred, ax=ax)
+    PARAMS = {'max_epochs': 3,
+              'LR': 0.02,
+              'batch_size': 32}
 
-    neptune_logger.experiment.log_image('confusion_matrix', fig)
+Step 3 - Define Lightning Module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This is minimal example of the ``pl.LightningModule``. Notice the Cross Entropy loss in the ``training_step`` method.
```

Comment:
Hmm, how is cross entropy important for me learning how to log things?

Reply:
It is the only metric that will be logged to Neptune. I added info that this will be logged to Neptune, so that user see where exactly logging happens.